source/geod24/LocalRest.d

```diff
@@ -921,9 +921,10 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
 
     ***************************************************************************/
 
-    public this (Listener!API listener, Duration timeout = 5.seconds)
+    public this (IncomingAPI) (Listener!IncomingAPI listener, Duration timeout = 5.seconds)
         @trusted nothrow
     {
+        static assert(is(IncomingAPI : API));
```

Comment:
Error message ?

Reply:
Updated, how about that?

Reply:
Works too, although static assert provides a better UX.